addons/website/models/res_lang.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+
+class Lang(models.Model):
+    _inherit = "res.lang"
+
+    @api.multi
+    def write(self, vals):
+        if vals.get('active') == False:
```

Comment:
I understand that you copy from base/res.company
but to avoid a next reader to replace it naively by
```python
 if not vals.get('active'):
```

can you write explicitly: 
```python
   if 'active' in vals and not vals['active']:
```

Reply:
done